api/src/main/java/com/spectralogic/tpfr/api/ServerServiceFactoryImpl.java

```diff
@@ -0,0 +1,73 @@
+package com.spectralogic.tpfr.api;
+
+import okhttp3.OkHttpClient;
+import okhttp3.Request;
+import okhttp3.logging.HttpLoggingInterceptor;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import retrofit2.Retrofit;
+import retrofit2.converter.simplexml.SimpleXmlConverterFactory;
+
+import java.util.Optional;
+import java.util.concurrent.TimeUnit;
+
+import static okhttp3.OkHttpClient.Builder;
+
+public class ServerServiceFactoryImpl implements ServerServiceFactory {
+
+    private final Logger LOG = LoggerFactory.getLogger(ServerServiceFactoryImpl.class);
+    private ServerService serverService;
+
+    @Override
+    public Optional<ServerService> getServerService() {
+        return Optional.ofNullable(serverService);
+    }
+
+    @Override
+    public ServerService createServerService(final String endpoint) {
+
+        LOG.info("Creating a server service to: " + endpoint);
```

Comment:
This isn't using the correct logging format of `LOG.info("message {}", variable)`